examples/maildev/README.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Maildev
+description: A simplest way to test your generated emails during development with an easy to use web interface.
+buttonSource: https://github.com/mittalyashu/railway-maildev/blob/main/readme.md
+tags:
+  - maildev
+  - nodejs
```

Comment:
Could you change this to `javascript` instead of `nodejs`? That's the pattern we have at the moment.

Reply:
sure